server/schema/dashboard.go

```diff
@@ -54,6 +55,7 @@ type DashboardElementView struct {
 
 	Realtime types.NullInt64   `json:"realtime"`
 	Servers  []newrelic.Server `json:"servers,omitempty"`
+	Tasks    []*teamcity.Build `json:"tasks,omitempty"`
```

Comment:
In this implementation the structure of teamcity.Build is larger than teamcity.Build and can have attachments. Also, due to poor library implementation, the SearchBuild method can return a large amount of data and this will be a large memory consumption if passed by value.  Bring everything to one kind, it's good and the code will be more "transparent".